sphinx/tutorial/cylc/scheduling/advanced-scheduling.rst

```diff
@@ -0,0 +1,73 @@
+Advanced Scheduling
+===================
+
+In this section we will quickly run through some of the more advanced features
+of cylc's scheduling logic.
+
+
+.. include:: ../../../hyperlinks.rst
+   :start-line: 1
+
+
+.. _tutorial-qualifiers:
+
+Qualifiers
+----------
+
+So far we have written dependencies like ``foo => bar``. This is, in-fact,
+shorthand for ``foo:succeed => bar``, it means that the task ``bar`` will run
+once ``foo`` has finished successfully. If ``foo`` were to fail then ``bar``
+would not run. We will talk more about these :term:`task states <task state>`
+in the TODO:Runtime_Section.
+
+We would refer to the ``:succeed`` as a :term:`qualifier`. There are qualifiers
+for different :term:`task states <task state>` e.g:
+
+``:start``
+    When the task has started running.
+``:fail``
+    Occurs when the task finishes if it fails (non-zero return code).
+``:finish``
+    Occurs when the task has stopped running irrespective of whether it has
+    succeeded or failed.
+
```

Comment:
It could be reworded slightly (e.g. what exactly 'occurs' when the task finishes/fails). Something like: "The dependency is met/satisfied when the task fails". Or: "the dependent task is triggered when the task fails"